test/parallel/test-crypto.js

```diff
@@ -46,7 +46,14 @@ assert.throws(function() {
   const context = credentials.context;
   const notcontext = { setOptions: context.setOptions, setKey: context.setKey };
   tls.createSecureContext({ secureOptions: 1 }, notcontext);
```

Comment:
Updated tests below to check for existence of new openSSLErrorStack property for when it should and should not be there